common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -6,7 +6,9 @@
 %>
 <form class="choicegroup capa_inputtype" id="inputtype_${id}">
     <fieldset ${describedby}>
-      <legend id="${id}-legend" class="response-fieldset-legend field-group-hd">${response_data['label']}</legend>
+      % if response_data['label']:
```

Comment:
I don't see anything wrong with legend without any value. Since we're not doing this in rest of the templates, so i think we can go without this condition.